libs/portal-integration-angular/src/lib/core/directives/loading-indicator.directive.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ComponentFactoryResolver,
+  ComponentRef,
+  Directive,
+  ElementRef,
+  Input,
+  OnChanges,
+  Renderer2,
+  SimpleChanges,
+  ViewContainerRef,
+} from '@angular/core';
+import { LoadingIndicatorComponent } from '../components/loading-indicator/loading-indicator.component';
+
+@Directive({
+  selector: '[onecxLoadingIndicator]',
+})
+export class LoadingIndicatorDirective implements OnChanges {
+  @Input() onecxLoadingIndicator = false;
+  @Input() overlayFullPage = false;
+  @Input() isLoaderSmall? = false;
+
+  private componentRef: ComponentRef<LoadingIndicatorComponent>;
+
+  constructor(
+    private viewContainerRef: ViewContainerRef,
+    private componentFactoryResolver: ComponentFactoryResolver,
+    private el: ElementRef,
+    private renderer: Renderer2
+  ) {}
+
+  ngOnChanges(changes: SimpleChanges) {
+    if (changes.onecxLoadingIndicator || changes.overlayFullPage) {
```

Comment:
change to
`if (changes['ocxLoadingIndicator'] || changes['overlayFullPage']) {`